TypeCobol.LanguageServer/Completion Factory/CompletionFactoryHelpers.cs

```diff
@@ -191,7 +191,7 @@ public static List<CompletionItem> CreateCompletionItemsForVariableSetAndDisambi
             return results;
         }
 
-        public static CompletionItem CreateCompletionItemForSingleVariable(DataDefinition parent, DataDefinition variable, TypeCobolOptions options, bool qualify)
+        public static CompletionItem CreateCompletionItemForSingleVariable(DataDefinition parent, DataDefinition variable, TypeCobolOptions options, bool qualify, TokenType qualificationTokenType = TokenType.OF)
```

Comment:
3 params revolving around the qualification style. Maybe it could be merged into a `QualificationStyle` enum param: `DoNotQualify, UseTypeCobolQualifiedNameSeparator, UseOF, UseIN` ? See how it goes with the callers. Keep current code if it creates more problems than it solves